samples/drivers/i2s/litex/sample.yaml

```diff
@@ -9,3 +9,9 @@ common:
       type: one_line
       regex:
         - "i2s example (.*)"
+
+tests:
+  sample.drivers.i2s.litex:
+    tags: I2S
+    platform_allow: litex_vexriscv
+    harness: TBD
```

Comment:
there's something wrong here, 	common defines a console harness which is overriden here with "TBD", can you explain the rationale?

Reply:
I realize I probably don’t understand this part. The problem I had was that CI ran twister on this directory and failed due to not finding any tests to run.

Reply:
`common` section should be merged to the test actually, there's a single entry. platform_allow + harness: console should be enough to avoid twister running the test I guess, @PerMac ?

Reply:
`harness: console` means that a test requires to get a line from `regex` entry in the connected output. `harness: TBD` is not recognized by twister and will make the test skip on every platform. `platform_allow` is also a filter, it will only allow test execution on `litex_vexriscv`. If this is in fact the only platform able to run this test then using `platform_allow` is the way to go. You shouldn't then use `harness: TBD`